.github/workflows/aro-hcp-dev-env-cd.yml

```diff
@@ -161,6 +168,14 @@
                 --parameters kvNames="['${SVC_KV_NAME}']" \
                 --parameters githubActionsPrincipalID=${{ secrets.GHA_PRINCIPAL_ID }}
 
+              # enable aks metrics
+              AZ_MONITOR_RESOURCE_ID=$(az deployment group show --resource-group "${REGIONAL_RESOURCEGROUP}" --name "metrics-infra-${GITHUB_RUN_ID} --output tsv --query properties.outputs.monitorId.value)
```

Comment:
How quick is `az deployment group show`? I think it could be refactored to one call if it is slow

Reply:
It is not slow but I agree it is redundant.  I'll need to dig into how it can be set once as a global so both steps have access to it.